WordPress/Classes/ViewRelated/Views/NoResultsViewController.swift

```diff
@@ -197,15 +201,15 @@ import Reachability
     }
 
     /// Public method to show a 'Dismiss' button in the navigation bar in place of the 'Back' button.
-    ///
-    func showDismissButton() {
+    /// Accepts an optional title, if none is provided, will default to 'Dismiss'
+    func showDismissButton(title: String? = nil) {
         navigationItem.hidesBackButton = true
 
-        let dismissButton = UIBarButtonItem(title: NSLocalizedString("Dismiss", comment: "Dismiss button title."),
+        let dismissButton = UIBarButtonItem(title: title ?? NSLocalizedString("Dismiss", comment: "Dismiss button title."),
```

Comment:
To reduce redundancy, maybe we can create the title once and use that in both places? 

Ex:
`let buttonTitle = title ?? NSLocalizedString("Dismiss", comment: "Dismiss button title.")`

Reply:
Totally